extension/jdtls.ext/com.microsoft.gradle.bs.importer/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,18 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Gradle Build Server Importer
+Bundle-SymbolicName: com.microsoft.gradle.bs.importer;singleton:=true
+Bundle-Version: 0.1.0
+Bundle-Activator: com.microsoft.gradle.bs.importer.Activator
+Bundle-RequiredExecutionEnvironment: JavaSE-11
```

Comment:
JavaSE-17?

Since build server is compiled to Java 17, the minimum JDK runtime should be 17.